.syft.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright The Enterprise Contract Contributors
```

Comment:
Technically, this should go after the `---` otherwise yaml parsers will think that this section is its own `null` yaml document.